joomla/joomla-cms
# [4.0] Replace highlighter javascript with mark.js

Pull Request for Issue # .

### Summary of Changes
Replace Joomla's own implementation of highlighting with mark.js


### Testing Instructions
Check the smart search in the front end, you should get:
<img width="594" alt="screen shot 2017-05-30 at 12 52 47" src="https://cloud.githubusercontent.com/assets/3889375/26578128/556d6d2e-4537-11e7-969f-2d0bdc3114b6.png">



### Expected result



### Actual result



### Documentation Changes Required
